src/lib/V3DutchOrderLib.sol

```diff
@@ -28,6 +28,8 @@ struct V3DutchOrder {
     V3DutchInput baseInput;
     // The tokens that must be received to satisfy the order
     V3DutchOutput[] baseOutputs;
+    // Adjust the startAmount to account for changes to gas
+    uint256 gasSnapshot;
```

Comment:
something more descriptive that specifies what unit this is in would be nice, like `baseFeeGwei `

Reply:
ah also I think this should be moved above the baseInput and baseOutputs, it's easier for parsing to have all of the statically sized elements before any dynamic ones IMO

Reply:
Thanks! I'll call it `baseFee` since it's the actual basefee denominated in wei.